### PR TITLE
Add saving and loading functionality

### PR DESCRIPTION
**Pull request by devfag**
You can save and load using the `/save` and `/load` commands. These commands will ask you for the name of a file in the saves directory (minus the `.json` extension). The game will either save the current story to the file or load the story from the file depending on the command used.

In addition, `/menu` and `/exit` ask the user if they want to save before proceeding, and there is now an additional option on the top-level menu giving the user the ability to start a game from a save.

In order to make loading games look nicer, I took the liberty of revising the story manager and giving it it's own print function, which properly formats and colors the loaded story as if the user had been playing since the beginning. The story manager now has two arrays: `results` and `actions`, instead of the previous `story`. This allows the information to be more easily parsed and outputted according to the existing display rules.

### CODE REVIEW

File: play.py [hunk 1]
```diff
@@ -165,6 +165,52 @@ def new_story(generator, prompt, context):
...
+            output("Please enter a valid savefile name. ", colors["error"])
+        else:
+            break
+    savefile = os.path.splitext(savefile.lstrip("saves/").strip())[0]
+    story.savefile = savefile
+    savedata = story.to_json()
+    Path("saves/").mkdir(parents=True, exist_ok=True)
+    with open("saves/" + savefile + ".json", 'w') as f:
+        f.write(savedata)
+        output("Successfully saved to " + savefile, colors["message"])
```
Comment (AIDungeonWiXAnon) [inline play.py]: I feel that this should have some sort of exception handling, since it's attempting to write to a file

Comment (devfag) [reply play.py]: Good point, will add

**Comment by devfag** [comment]
Sneaking in one more commit that adds the ability to turn off color and wrapping on the `/print` command

**Comment by devfag** [comment]
Found a bug: don't merge yet

**Comment by devfag** [comment]
Two things:
1. My new loading method was causing `story` to be referenced before being assigned when entering a custom prompt & context
2. Loading prompts/contexts from a file seems to reverse the prompt and the context

**Comment by devfag** [comment]
@cloveranon Can you clarify if on your fork, the `context` is the permanent story context and the `prompt` is the initial story prompt or vice-versa? There seems to be some confusion between the two in the current codebase.

**Comment by devfag** [comment]
My proposal would be to have `context` be the permanent one, and `prompt` be the initial incident. Then, in the pastebin stories that only have one line, use it as the `prompt`, since they're already worded to be open-ended most of the time.

**Comment by devfag** [comment]
Fixed all known issues with this PR. Please do a final review (it's a big one) and merge when you're ready

**Pull request merged**